investigation-service/src/main/resources/application.yaml

```diff
@@ -4,7 +4,6 @@ spring:
       topic-name: nbs_Public_health_case
     output:
       topic-name-reporting: nrt_investigation
```

Comment:
Are we removing the elastic search topic for optimization?

Reply:
I've never seen any realization for it in this version. The topic name was never used.

Reply:
Hmm I believe ES outputs were originally for the patient, provider and organization search. There isn't a requirement for the Investigation afaik. It can be taken up in later sprints when the requirement arises.